code/modules/organs/body_modifications.dm

```diff
@@ -50,7 +50,7 @@ var/global/list/modifications_types = list(
 
 /datum/body_modification/proc/is_allowed(var/organ = "", datum/preferences/P)
 	if(!organ || !(organ in body_parts))
-		usr << "[name] isn't useable for [organ_tag_to_name[organ]]"
+		//usr << "[name] isn't useable for [organ]"
```

Comment:
Why that was commented out?

Reply:
on spawn it always outputs this two times because some organs cant be modified

Reply:
But why that organs try to be modified?

Reply:
augmentation